### DISCUSSION TITLE
Align label X axis

### DISCUSSION THREAD

**Discussion by mikelbd** [patrykandpatrick/vico#468] [Questions]
Hi, I would like to know if there is a way to align the label with the vertical bars.

<img width="421" alt="image" src="https://github.com/patrykandpatrick/vico/assets/147382482/ab9f7c88-5785-41e0-96a5-a1115cc1bb95">

**Comment by patrickmichalik** [comment]
Hello! Could you please provide some extra context? We’re not entirely sure what the image represents and how it relates to Vico. Are you looking to implement a chart with such columns and alignment options?

**Reply by mikelbd** [reply]
Correct, it would be to be able to configure a graph with the possibility of aligning the columns to right left or center as in the image, at the moment all the labels I manage to put them centered, but I do not see the possibility of aligning to left or right.

**Reply by patrickmichalik** [reply]
Got it. This isn’t possible yet, but we’ll be looking into adding such functionality. Please see #428.
